labchecker/plugins/nodepool/flavor.py

```diff
@@ -0,0 +1,25 @@
+from labcheck import Plugin
+import commands
+import yaml
+
+
+class FlavorPlugin(Plugin):
+    ptype = 'nodepool'
+    name = 'flavor'
+
+    def check(self):
+        self.failed = False
+        self.reasons = []
+        sg = 'openstack --os-cloud %s flavor list -f yaml -c VCPUs -c RAM -c Name' % self.cloud
```

Comment:
I think we should output `DISK` size of the flavor we find, that impact whether we use boot from image or volume instance.

Reply:
```
[nodepool] flavor----------------------- PASSED
Info:
- Flavor 4U8G: not found
- Flavor 8U8G: ['v2-highcpu-8 (0G)', 'v1-standard-8 (200G)']
- Flavor 1U2G: not found
- Flavor 2U2G: ['v2-highcpu-2 (0G)']
```
ok, will add disk info after flavor name, like above